src/pages/analyses.js

```diff
@@ -1,12 +1,5 @@
 import React from "react";
-import Navigation from "../components/navigation/Navigation";
-import NavigationConstants from "../components/navigation/NavigationConstants";
 
 export default function Dashboard() {
```

Comment:
Should this be renamed?
```suggestion
export default function Analyses() {
```